tests/Nox.Tests/DatabaseIntegrationTests/Models/Entities.TestEntityForTypes.cs

```diff
@@ -94,6 +94,11 @@ public partial class TestEntityForTypes : AuditableEntityBase
     /// </summary>
     public Nox.Types.CurrencyNumber? CurrencyNumberTestField { get; set; } = null!;
 
+    /// <summary>
+    ///  (Optional).
+    /// </summary>
+    public Nox.Types.DateTime? DateTimeTestField { get; set; } = null!;
```

Comment:
hm, why has this changed? I think nothing was changes in definition

Reply:
someone is not generating the code when changing the definition